.github/codeql/codeql-config.yml

```
@@ -0,0 +1,10 @@
+name: "CodeQL config"
+queries:
+  - name: Run custom javascript queries
+    uses: ./.github/codeql/queries
```

Make sure we keep running the standard queries too (and we might as well run the security+quality set instead of just security).
```suggestion
  - name: Run standard queries
    uses: security-and-quality
  - name: Run custom javascript queries
    uses: ./.github/codeql/queries
```